source/ProgArgs.h

```diff
@@ -619,6 +644,15 @@ class ProgArgs
         bool getRunS3AclGet() const { return runS3AclGet; }
         bool getRunS3BucketAclPut() const { return runS3BucketAclPut; }
         bool getRunS3BucketAclGet() const { return runS3BucketAclGet; }
+        bool getRunS3StatDirs() const { return runS3StatDirs; }
+
+        bool getRunS3GetObjectMetadata() const { return s3ObjectMetadataRequested() && runStatFilesPhase; }
```

Comment:
now that we have separate phases for s3 bucket/object metadata operations, there should no longer be a need to bind this to other phases. 
e.g. here the get obj metadata ops can run fine even if `runStatFilesPhase` is not selected by the user.